### PR TITLE
Add version badges

### PR DESCRIPTION
**Pull request by RobertLK**
Show last version and master status on README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +9,8 @@ This repo contains the tools and step by step instructions so that your users ca
 
 1) [References](#references) 
 2) [Requirements](#requirements)
-3) [Installing the SDK](#installing-the-sDK)
-4) [Profile Retrieval](#profile-retrieval)
-5) [Handling Users](#handling-users)
+3) [Installing the SDK](#installing-the-sdk)
+4) [Configuration](#configuration)
 6) [Running the Example](#running-the-example)
```
Comment (eduardoyoti) [inline README.md]: Aren't we missing the 5) ?

File: README.md [hunk 2]
```diff
@@ -41,7 +42,7 @@ For more information please follow our developer page instructions located [here
 Make sure you have mavenCentral repository added in your allProjects entry in your root build.gradle:
 
 ```gradle
-allprojects {
+allprojects {Some sections were change
```
Comment (eduardoyoti) [inline README.md]: Misplaced text?

File: README.md (hunk 1 above)
Comment (echarrod) [reply README.md]: Don't mean to butt in, but you can change them all to `1)`, and the numbers are auto-updated 🙂

Comment (echarrod) [reply README.md]: See https://github.com/getyoti/yoti-dotnet-sdk/blob/master/README.md (and [raw markdown](https://raw.githubusercontent.com/getyoti/yoti-dotnet-sdk/master/README.md))

Comment (RobertLK) [reply README.md]: Yeah, I figured Markdown doesn't care about the numbers but changing them all to 1 seems a bit extreme

File: README.md (hunk 2 above)
Comment (RobertLK) [reply README.md]: Yeah, not sure what happened there

File: README.md (hunk 1 above)
Comment (echarrod) [reply README.md]: But then you don't ever have to worry about updating numbers if you add/remove list items in the future 😃

Comment (RobertLK) [reply README.md]: Actually, we had a similar discussion at https://github.com/getyoti/yoti-doc-scan-android/pull/12 and decided we should just remove the numbers :grinning:

**Comment by RobertLK** [comment]
Merged into #15

**Pull request closed without merging**